### PR TITLE
kill loaders

### PR DESCRIPTION
**Pull request by nickpell**
The tests for this section of the code dependent on the ```*_attachment``` methods, but is unused in HQ, so I'm killing it.

### CODE REVIEW

File: couchdbkit/version.py [hunk 1]
```diff
@@ -4,5 +4,5 @@
 # See the NOTICE for more information.
 
 from __future__ import absolute_import
-version_info = (0, 9, 2)
+version_info = (0, 9, 3)
```
Comment (kkamalov) [inline couchdbkit/version.py]: why not use semver? @nickpell

**Comment by emord** [comment]
Using semver seems legit. maybe we should start putting these as beta builds for 1.0.0?

Any reason you aren't removing all attachment code here?

**Comment by nickpell** [comment]
> Any reason you aren't removing all attachment code here?

Just being conservative.  Planning on moving all attachment code in the next point release, after this PR has been deployed in production.

File: couchdbkit/version.py (hunk 1 above)
Comment (nickpell) [reply couchdbkit/version.py]: Hi @kkamalov, thanks for the feedback.  This fork of couchdbkit is intended for internal Dimagi use only, so we don't attempt to rigorously follow semantic versioning.  If you know of any other projects that use this fork, please let us know! AC

**Pull request merged**